app/services/errors/external_record_not_found_exception.rb

```diff
@@ -0,0 +1,2 @@
+class ExternalRecordNotFoundException < StandardError
```

Comment:
This is not the correct way to define custom errors because it's not following the path standards of rails, if your error is in an `errors` folder then the correct way to define it would be something like this:
```
class Errors
  class ExternalRecordNotFound < StandardError
  end
end
```

And you can invoke that error by using something like this: `Errors::ExternalRecordNotFound` 

Beware of the naming of the class, it's not necesary to define the Exception subfix